apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2

```diff
@@ -56,6 +56,8 @@ spec:
                 secretKeyRef:
                   name: newrelic-secrets
                   key: NEW_RELIC_LICENSE_KEY
+            - name: NODE_ENV
+              value: "production"
```

Comment:
`git grep NODE_ENV` in the kumascript repo doesn't turn up anything, so this seems safe, and a good catch to add this now for the future.